packages/beacon-node/src/chain/blocks/importBlock.ts

```diff
@@ -334,12 +333,6 @@ export async function importBlock(
     this.logger.verbose("After importBlock caching postState without SSZ cache", {slot: postState.slot});
   }
 
-  if (parentEpoch < blockEpoch) {
```

Comment:
still need this code snippet. Epoch transition normally happens at last 1/3 of last slot of epoch and we may have incorrect head there, so the `shufflingCache.build()` may not be the one to be used
this is the real on-chain block, we need to keep it

Reply:
I'm a bit confused.  If for some reason the chain reorgs after the first `build` call wouldn't the epoch transition happen again with the new block?  If that is the case it would call build from within the follow-up epoch transition right?

I can add this but this seems incorrect so before I push the commit I'll leave it here for your review
```ts
if (computeEpochAtSlot(parentBlockSlot) < blockSlot) {
    const shuffling = postState.epochCtx.shufflingCache?.getSync(
      postState.epochCtx.nextEpoch,
      postState.epochCtx.nextDecisionRoot
    );
    if (!shuffling) {
      postState.epochCtx.shufflingCache
        ?.build(
          postState.epochCtx.nextEpoch,
          postState.epochCtx.nextDecisionRoot,
          postState,
          postState.epochCtx.nextActiveIndices
        )
        .then((shuffling) => {
          postState.epochCtx.nextShuffling = shuffling;
        })
        .catch((err) => {
          this.logger.error(
            "Error building next shuffling in importBlock",
            {
              epoch: postState.epochCtx.nextEpoch,
              decisionRoot: postState.epochCtx.nextDecisionRoot,
            },
            err
          );
        });
    }
  }
```

Reply:
as discussed offline, we don't need it because whatever happens, epoch transition should run and build and populate ShufflingCache so no need this anymore